src/resolve/meson.build

```diff
@@ -131,12 +131,16 @@ systemd_resolved_sources += custom_target(
 systemd_resolved_dependencies = [threads, libgpg_error, libm]
 if conf.get('ENABLE_DNS_OVER_TLS') == 1
         if conf.get('DNS_OVER_TLS_USE_GNUTLS') == 1
-                systemd_resolved_sources += files('resolved-dnstls-gnutls.c',
-                                                  'resolved-dnstls-gnutls.h')
+                systemd_resolved_sources += files('''
+                        resolved-dnstls-gnutls.c
+                        resolved-dnstls-gnutls.h
+                '''.split())
```

Comment:
As in the other place... I think we might want to move in the opposite direction.

Reply:
Done.